challenge1Week2Objects.js

```diff
@@ -0,0 +1,7 @@
+var a;
+var b;
+
+for (a = 1; a < 16; a++) {
+    b = a * a;
+    console.log(a, b);

```

Comment:
Good work on this but please you are not creating an object anywhere. 

How do we create an empty object in JavaScript?
One way to do it in js is
```js
var myNewObject = {}
```

How do we add key-value pairs to objects?
I will cover a case where we first create an object and add key-value pairs to it from variables since its what you need to tackle this. We can talk about the rest later.
```js
// given an empty object below
var person = {}

// and two variables we want to use as a key-value pair below
var myKey = "age"
var myValue = 13

// we can add the key value pair to the object like
person[myKey] = myValue

console.log(person) // logs {age: 13} to the console
```

>So for each and every number in the loop, add that number and the key to the object and its square as the value